tools/functional-tester/etcd-tester/stresser.go

```diff
@@ -165,16 +165,11 @@ func (s *stresser) Stress() error {
 	if err != nil {
 		return fmt.Errorf("%v (%s)", err, s.Endpoint)
 	}
-	ctx, cancel := context.WithCancel(context.Background())
 
 	wg := &sync.WaitGroup{}
 	wg.Add(s.N)
-
-	s.mu.Lock()
```

Comment:
apparently, stress() will be called serially not concurrently. so no need to use lock()